nix/local/envs.nix

```diff
@@ -33,16 +34,33 @@ in {
     name = "Cardano JS SDK Local Env";
     imports = [formattingModule];
 
+    packages = with inputs.nixpkgs; [
+      awscli2
+      python310
```

Comment:
python?

Reply:
It was present in dapps, a left over from trying to align those, I’ll remove it

Reply:
Fixed in f8ffbac1bc0a35b3aa17df904f501f43ebd9632e